doc/man1/openssl-pkcs12.pod.in

```diff
@@ -362,7 +362,9 @@ to be needed to use MAC iterations counts but they are now used by default.
 
 =item B<-nomac>
 
-Don't attempt to provide the MAC integrity.
+Don't attempt to provide the MAC integrity. This can be useful with FIPS
```

Comment:
can be useful with **_the_** FIPS provider.

Reply:
fixup pushed, still OK?